### PR TITLE
Command History/ Auto complete added

### PR DESCRIPTION
**Pull request by matthewoconnor-development**
#13 Hopefully should be ok now after rebase.

### CODE REVIEW

File: shell.js [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
+* OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+* DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,
+* INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+* (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+* SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+* HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+* STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+* IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*/
```
Comment (mcollina) [inline shell.js]: would you mind not changing the formatting?

File: shell.js [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
 var CliTable = require('cli-table');
 require('colors');
 
 
 
 /**
- * fuge run -> executes through here - runs all and starts a log tail
- * hitting any key will stop the tail
- * need a command history
- */
```
Comment (mcollina) [inline shell.js]: also here

File: shell.js [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
+*/
 module.exports = function() {
   var _config;
   var _runner;
   var _proxy;
+  var vorpal =  new Vorpal();
   var tableChars = { 'top': '' , 'top-mid': '' , 'top-left': '' , 'top-right': '',
-                     'bottom': '' , 'bottom-mid': '' , 'bottom-left': '' , 'bottom-right': '',
-                     'left': '' , 'left-mid': '' , 'mid': '' , 'mid-mid': '',
-                     'right': '' , 'right-mid': '' , 'middle': ' ' };
```
Comment (mcollina) [inline shell.js]: and here

File: README.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ The fuge shell supports the following commands:
 * **tail [process] | all** - tail output for a process or for all processes
 * **untail [process] | all** - end tail output for a specific processes or for all processes
 * **grep 'search string' [process]** - searches a processes log or all processe logs
-* **exit** - termiate all managed process and exit
+* **quit** - termiate all managed process and exit
```
Comment (mcollina) [inline README.md]: would it be a big issue supporting exit as well?

**Comment by matthewoconnor-development** [comment]
I didn't change those, Wyatt did. I just picked it up where he left off. Exit cant be used as its a vorpal keyword and throws an error, hence the change to quit.

**Comment by mcollina** [comment]
Given that exit is just asking for a confirmation, I would keep exit and remove quit. I don't see the need of two things for doing the same stuff.

**Comment by mcollina** [comment]
Why this was closed?

**Pull request closed without merging**